Tests/RevenueCatUITests/PurchaseCompletedHandlerTests.swift

```diff
@@ -35,13 +36,17 @@ class PurchaseCompletedHandlerTests: TestCase {
             .onPurchaseStarted {
                 started = true
             }
+            .onPurchaseStarted { package in
```

Comment:
Does this work for a variable name?! Even though `package` is a reserved word? 😅